inlineplz/linter_runner.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=ungrouped-imports,broad-except,too-many-instance-attributes
+"""Linter configurations."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import asyncio
+import fnmatch
+import multiprocessing
+import os.path
+import shutil
+import sys
+import time
+import traceback
+
+from identify import identify
```

Comment:
i'm surprised prospector didnt yell at you to separate this 3rd party imports from the internal imports

Reply:
fixed